client/src/components/Login.js

```diff
@@ -55,113 +56,147 @@ const useStyles = makeStyles(theme => ({
 function Login() {
   const classes = useStyles();
 
-  return (
-    <Grid container component="main" xs={12} sm={8} className={classes.root}>
-      <CssBaseline />
-      <Grid
-        item
-        xs={false}
-        sm={6}
-        md={5}
-        className={classes.image}
-        style={{ display: "flex", justifyContent: "flex-start" }}
-      >
-        <div
-          style={{
-            display: "flex",
-            alignItems: "center",
-            justifyContent: "center",
-            flexDirection: "column",
-            height: "150px"
-          }}
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
+  const handleChange = event => {
+    if (event.target.name === "email") {
+      setEmail(event.target.value);
+    }
+    if (event.target.name === "password") {
+      setPassword(event.target.value);
+    }
+  };
+  const signIn = () => {
+    signApi(email, password)
```

Comment:
@mhallaq all the changes look great! 
I had a look at the problem we discuss. What you need to do to avoid the page getting reloaded is to prevent the event from it's default behaviour. Because the submit button on a form usually will POST the form data by reloading the page. But since we need to post using fetch you can just prevent that from happening by doing that 
```
const signIn = (event) => {
    e.preventDefault();
   .....
``` 

Have a look [on that](https://stackoverflow.com/questions/50193227/basic-react-form-submit-refreshes-entire-page) for more info.